src/modules/image-upload/ImageUploadHelper.js

```diff
@@ -2,24 +2,31 @@
 
 import { bus } from '../../core-client';
 
-export type UploadOptions = {
+type UploadOptions = {
+	filename: string;
 	uploadType: 'content';
 	generateThumb?: boolean;
 	textId: string;
 } | {
+	filename: string;
 	uploadType: 'avatar';
 	generateThumb?: boolean;
 }
 
 type UploadPolicy = {
-	keyPrefix: string;
-	bucket: string;
-	acl: string;
-	policy: string;
-	'x-amz-algorithm': string;
-	'x-amz-credential': string;
-	'x-amz-date': string;
-	'x-amz-signature': string;
+	request_url: string;
+	original: string;
+	thumbnail: string;
+	policy: {
```

Comment:
rename to uploadParams or uploadArgs (or just params/args). Dislike uploadPolicy.policy.policy.